code/datums/datacore.dm

```diff
@@ -186,42 +186,27 @@
 
 /datum/datacore/proc/get_manifest()
 	var/list/manifest_out = list()
-	var/list/dept_list = list(
-		"Command" = DEPT_BITFLAG_COM,
-		"Very Important People" = DEPT_BITFLAG_VIP,
-		"Security" = DEPT_BITFLAG_SEC,
-		"Engineering" = DEPT_BITFLAG_ENG,
-		"Medical" = DEPT_BITFLAG_MED,
-		"Science" = DEPT_BITFLAG_SCI,
-		"Supply" = DEPT_BITFLAG_CAR,
-		"Service" = DEPT_BITFLAG_SRV,
-		"Civilian" = DEPT_BITFLAG_CIV,
-		"Silicon" = DEPT_BITFLAG_SILICON
-	)
+	var/static/list/heads = make_associative(GLOB.command_positions + list(JOB_NAME_QUARTERMASTER))
```

Comment:
remove quartermaster.

Reply:
Hm, yeah, this isn't needed now anyways, given that the manifest is now sorted by job anyways, and QM is above all cargo.